### PR TITLE
Adding initial support for ARIA 1.1 roles: none, cell, table, searchb…

### PR DESCRIPTION
**Pull request by yzen**
…ox, text.

### CODE REVIEW

File: lib/commons/aria/index.js [hunk 1]
```diff
@@ -194,24 +212,24 @@ lookupTables.role = {
...
+		attributes: {
+			allowed: ['aria-colindex', 'aria-colspan', 'aria-rowindex', 'aria-rowspan']
 		},
 		owned: null,
 		nameFrom: ['author', 'contents'],
-		context: null,
-		implicit: ['input[type="checkbox"]']
+		context: ['row']
 	},
-	'switch': {
```
Comment (yzen) [inline lib/commons/aria/index.js]: I just moved switch down to preserve the alphabetical order in this file. Hope that's OK.

**Comment by yzen** [comment]
I was also wondering, if this PR is good, what your process around bumping the axe-core versions (in terms of using it as an npm module)? Thanks

**Comment by dsturley** [comment]
There is one more issue that needs to be in 1.1.0; #67.  When it will be available is a question for @dylanb 

Thanks again for your contribution.

**Comment by yzen** [comment]
Ah good to know, thanks, I'll keep an eye on it.

**Comment by dylanb** [comment]
@yzen full disclosure - we are still working on how/when and how frequently we release the modules that are dependent on this repo...

**Pull request merged**